MainMenu Scene

MariuszAKS opened the pull request:
Added Main Menu Scene.
- Buttons and sliders
- Slider values updating text next to them
- New Simulation right now just loads GameScene Renamed UIScene na GameScene.
Added scenes to build settings and set indexes.

MariuszAKS committed:
MainMenu Scene

Added Main Menu Scene.
- Buttons and sliders
- Slider values updating text next to them
- New Simulation right now just loads GameScene
Renamed UIScene na GameScene.
Added scenes to build settings and set indexes.

Michal-Roman0 reviewed (commented):
Zawija się tekst w niektórych miejscach menu na 16:9 aspect

MariuszAKS committed:
MainMenu NewSimulation Functionality

(Almost) All inputs in Main menu do something:
- Sliders work and have inpact on Organism Stats
- Sliders for number and proportion dictate the amount of organisms spawned
- After clicking New Simulation, it goes to SimulationScene, generates organisms, places them randomly on the board and sets their stats
- Only load simulation doesn't have functionality yet

MariuszAKS committed:
Changed MainMenu a bit (fixed text wrapping)

Hopefully it looks good now

MariuszAKS committed:
Changed labels font-size in MainMenu

MariuszAKS committed:
Tweaked MainMenu UI, added Save button

xAdiro reviewed (approved):
Po poprawce wygląda prawidłowo
